testsuite/shared/src/main/java/org/jboss/as/test/integration/management/extension/customcontext/testbase/CustomManagementContextTestBase.java

```diff
@@ -22,8 +22,7 @@
 import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.ROLLBACK_ON_RUNTIME_FAILURE;
 import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.SUBSYSTEM;
 import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.SUCCESS;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```

Comment:
Our code style is not to use wildcard imports.

Reply:
Sorry, I added AssertFalse and IDEA decided to group all Assert imports into * import. Will take better care of this next time